core/src/main/java/com/linecorp/armeria/common/logging/RequestLogBuilder.java

```diff
@@ -42,7 +42,13 @@
 
     /**
      * A dummy {@link RequestLogBuilder} that discards everything it collected.
+     *
+     * @deprecated This field will be removed without a replacement. The {@link NoopRequestLogBuilder}
+     *             means that it does not perform any operations on {@link RequestLogBuilder}.
+     *             The {@link NoopRequestLogBuilder} behavior can be achieved by performing operations
+     *             on {@link RequestContext#logBuilder()} if {@link RequestContext} is not null.
```

Comment:
`NoopRequestLogBuilder` is not part of public API, so it's not a good idea to add a link to it in public API Javadoc. I also think the proposed explanation here is only valid in the context of this pull request, i.e. Users will have no idea why you're saying this. It has to be explained in our users' standpoint. How about:

```
This field will be removed without a replacement. Use {@link DefaultRequestLog} or a mock.
```

Reply:
Thanks for reviewing! I will address it. 🙏